spec/lib/email_sponsees_extractor_spec.rb

```diff
@@ -50,27 +53,28 @@
 
   context 'Regression tests' do
     it 'Multipart message' do
-      sponsees = test_case 'email-sponsor-multipart'
+      test_case 'email-sponsor-multipart'
       expect(sponsees.first).to eq('07123456789')
     end
 
     it 'Multiple levels of multipart messages' do
-      sponsees = test_case 'email-sponsor-multilevel-multipart'
+      test_case 'email-sponsor-multilevel-multipart'
       expect(sponsees.first).to eq('example.user2@example.co.uk')
     end
 
     it 'Base64 encoded message' do
-      sponsees = test_case('email-sponsor-base64')
+      test_case('email-sponsor-base64')
```

Comment:
I find the inconsistent use parentheses mildly upsetting

Reply:
Yeah, that is odd.